whisper/tokenizer.py

```diff
@@ -127,21 +129,47 @@
 }
 
 
-@dataclass(frozen=True)
+@dataclass
 class Tokenizer:
     """A thin wrapper around `GPT2TokenizerFast` providing quick access to special tokens"""
```

Comment:
```suggestion
    """A thin wrapper around `tiktoken` providing quick access to special tokens"""
```

Reply:
I have also implemented whisper with tiktoken, but you have already released yours :) So I make just a few comments.